src/App.js

```diff
@@ -1,22 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+import GreetingsComponent from './components/GreetingsComponent';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>Refresh for a new greeting</h1>
+        <GreetingsComponent />
       </header>
```

Comment:
Since this milestone requires that you use Redux to fetch your API data. I tested the `Greetings.js` component instead of the second component you created `GreetingsComponets.jsx` which you are currently rendering.

- The  `GreetingsComponets.jsx` component does not adhere to the project requirements since it does not use Redux, kindly render the `Greetings.js` instead.

After rendering the `Greeting.js` component I encountered an error on the page.

![image](https://github.com/zamcham/hello-rails-front-end/assets/38088818/4bfaa625-6618-4656-af9c-d5b6c4f0fa1c)

- This is because the `index.js` is not wrapped in a `<Provider>` component which is responsible for making the Redux available to all your components. I can see that you implemented the code in the `main.js` file. Note that the entry point of the application is the `index.js` file therefore the `main.js` file is not recognized by the application making it unnecessary in your codebase.
- I highly recommend that you move all the code in the `main.js` file to the `index.js` file and delete the `main.js` file since it is redundant.

Otherwise, after implementing the suggested changes the `Greeting.js` component works perfectly fine. Great job!!! 🚀 

![image](https://github.com/zamcham/hello-rails-front-end/assets/38088818/ab6828e8-2d4f-48a4-9812-d3ce8e2e9888)